ingest/src/main/java/com/microsoft/azure/kusto/ingest/QueuedIngestClient.java

```diff
@@ -87,38 +87,39 @@ public IngestionResult ingestFromBlob(BlobSourceInfo blobSourceInfo, IngestionPr
                     ingestionProperties.getDatabaseName(), ingestionProperties.getTableName());
             String urlWithoutSecrets = SecurityUtils.removeSecretsFromUrl(blobSourceInfo.getBlobPath());
             if (blobSourceInfo.getRawSizeInBytes() > 0L) {
-                ingestionBlobInfo.rawDataSize = blobSourceInfo.getRawSizeInBytes();
+                ingestionBlobInfo.setRawDataSize(blobSourceInfo.getRawSizeInBytes());
```

Comment:
I get it's "the java way", but aren't you the one that said ".append" is less readable than "+="? 
Then surely "set" is less readable than "="?

On the other hand, if there was a time to make this change it's now, let's discuss.

Reply:
I am!

But here this is about OO fundamentals, which trump readability any day.